### PR TITLE
Managed to finished all the tasks as well as pass all the tests Chapter/Homework/3

### PR DESCRIPTION
**Pull request by Sakariya-Mohumed**
Initially had trouble with this task as I didn't know "Testing" needed to be inputted in the function in order for the tests to pass. I also duplicated the parsing process for the float which caused testing to fail but I managed to pull through.

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+namespace BootCamp.Chapter
+{
+    internal class Lesson3
+    {
+        public static void Demo()
+        {
+        }
+
+        public static float BMICalculator(float weightcal, float heightcal)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: method names should be verbs not nouns. Change this to `CalculateBmi`

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        {
+            var heightmeter = heightcal;
+            var heightsquared = heightmeter * heightmeter;
+            var totalbmi = weightcal / heightsquared;
+            Console.WriteLine("Your bmi is: " + totalbmi);
+            return totalbmi;
+        }
+
+        public static float BMiRequests(string prompt)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Rename to verb

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        {
+            Console.WriteLine(prompt);
+            var bm = Console.ReadLine();
+            var bm_1 = float.Parse(bm);
+
+            return bm_1;
+        }
+
+        public static int ageRequests(string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Rename to verb

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+            return ageconv;
+        }
+
+        public static string StringRequests(string messagestring)
+        {
+            Console.WriteLine(messagestring);
+            var stringinput = Console.ReadLine();
+
+            return stringinput;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Rename to verb

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            var bm_1 = float.Parse(bm);
+
+            return bm_1;
+        }
+
+        public static int ageRequests(string message)
+        {
+            Console.WriteLine(message);
+            var input = Console.ReadLine();
+            var ageconv = int.Parse(input);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: keep it simple and stupid (KISS): age

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+            var totalbmi = weightcal / heightsquared;
+            Console.WriteLine("Your bmi is: " + totalbmi);
+            return totalbmi;
+        }
+
+        public static float BMiRequests(string prompt)
+        {
+            Console.WriteLine(prompt);
+            var bm = Console.ReadLine();
+            var bm_1 = float.Parse(bm);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: keep it simple and stupid (KISS):
bmi = float.Parse(console.ReadLine());

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+            var input = Console.ReadLine();
+            var ageconv = int.Parse(input);
+
+            return ageconv;
+        }
+
+        public static string StringRequests(string messagestring)
+        {
+            Console.WriteLine(messagestring);
+            var stringinput = Console.ReadLine();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: keep it simple and stupid (KISS): input

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        public static void Demo()
+        {
+        }
+
+        public static float BMICalculator(float weightcal, float heightcal)
+        {
+            var heightmeter = heightcal;
+            var heightsquared = heightmeter * heightmeter;
+            var totalbmi = weightcal / heightsquared;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: keep it simple and stupid (KISS): bmi

**Review by Almantask — CHANGES_REQUESTED** [review]
Try to name things a bit better:
- variables should be camelCase
- method names should be PascalCase
- methods are actions- they should be named after verbs

**Comment by Sakariya-Mohumed** [comment]
I have edited the code and thank you very much for the feedback

**Review by Almantask — APPROVED** [review]

**Comment by Almantask** [comment]
Looks good 😄

**Pull request closed without merging**